compiler/src/main/scala/ingraph/compiler/cypher2gplan/GPlanResolver.scala

```diff
@@ -207,7 +207,7 @@ object GPlanResolver {
         case expr.PropertyAttribute(name, elementAttribute, _) => expr.PropertyAttribute(name,
           // see "scoped name resolver shorthand" above
           elementAttribute.transform(expressionNameResolver(snr)).asInstanceOf[expr.ElementAttribute],
-          snr.resolve(s"${elementAttribute.name}$$${name}", rn))
+          snr.resolve(s"${elementAttribute.name}.${name}", rn))
```

Comment:
@szarnyasg
Is there a specific case when `.` must be escaped?
It seems to me that removing the escaping doesn't cause any problem.